node/index.ts

```diff
@@ -17,10 +17,10 @@ const NINE_SECONDS_MS = 9 * 1000
 
 // Segments are small and immutable.
 const segmentCache = new LRUCache<string, Cached>({ max: 1000 })
-const searchCache = new LRUCache<string, Cached>({ max: 3000 })
+const searchCache = new LRUCache<string, Cached>({ max: 6000 })
```

Comment:
Since we recently made the pod bigger we can  safely increase the size of this cache because we have a lot of room to grow.